src/workers/notifications/postAdded.ts

```diff
@@ -66,6 +66,16 @@ const worker: NotificationWorker = {
             } as NotificationPostContext & Partial<NotificationDoneByContext>,
           }),
         );
+
+        const posts = await con.getRepository(SharePost).findBy({
+          authorId: post.authorId,
+        });
```

Comment:
We do need to evaluate whether the use already accepted notifications right?
Else the'll get a pointless notification added.